apps/btc_family/btc_helpers.c

```diff
@@ -103,7 +103,7 @@ int btc_get_segwit_addr(const uint8_t *public_key,
                         uint8_t key_len,
                         const char *hrp,
                         char *address) {
-  uint8_t rip[RIPEMD160_DIGEST_LENGTH] = {0};
+  uint8_t rip[SHA3_256_DIGEST_LENGTH] = {0};
```

Comment:
Please add a comment for why buffer size has been increased but smaller size is used.

Reply:
this popped up in the compiler warning. buffer overflow might happen here.

unsure why it did not appear in the original change.

Reply:
```suggestion
  // output of Hash160 (sha256 + ripemd160), use size of bigger hash
  uint8_t rip[SHA256_DIGEST_LENGTH] = {0};
```